src/argus/incident/ticket/dummy.py

```diff
@@ -1,5 +1,15 @@
+"""
+Minimal ticket creation plugin
+
+Serves as an example and is also used for testing.
+
+Inspired by django.core.mail.outbox, see https://docs.djangoproject.com/en/3.2/topics/testing/tools/#email-services
+django.core
```

Comment:
Is this line supposed to be here? Or just a copy paste error?

Reply:
You've marked what looks like 4 lines. I can only see the top three lines in this browser. Do you mean what is on line 7?

Reply:
Just line 7

Reply:
Thx. Yeah, line 7 was a copy paste error.